### ISSUE TITLE
Symbols >= and <= don't work

### ISSUE THREAD

**Issue by jamesattewell** [m-peko/booleval#57]
Hi @m-peko. 

I have run into an issue which I am struggling to solve at the moment. The symbols >= and <= are not working whereas GEQ and LEQ are working so its something to do with the parsing as it seems to think they are > or < and the expression fails as not valid.

I will continue looking and trying to solve it but wanted you to know there is an issue there.

Cheers

James

**Comment by jamesattewell** [comment]
The problem is in split_range but a fix for it eludes me at the moment, I think it has something to do with the delims that are passed into it as these contain < and >

**Comment by m-peko** [comment]
Hi @jamesattewell, I have fixed this issue. The root cause was in the `tokenizer::tokenize` function because we were splitting expression by single char symbols (like `<`, `>`, `(` etc.) and by whitespace. Therefore, symbols like `<=` and `>=` weren't recognized at all. I changed splitting expression to be by only parentheses symbols and by whitespace... Now it works fine and I have added some more test cases regarding symbol evaluation.
